Makefile

```
@@ -135,14 +132,19 @@ build-release-image-jupyter: pull-release-image-jupyter
 	@ echo "Building Jupyter Notebook release image..."
 
 	docker build \
-		--cache-from=$(docker_release_image_jupyter_repository):latest \
 		--file="$(CURDIR)/docker/jupyter/Dockerfile" \
 		--tag=$(docker_release_image_jupyter_repository):$(docker_image_version) \
 		--tag=$(docker_release_image_jupyter_repository):latest \
 		--build-arg="JUPYTER_NOTEBOOK_IMAGE_REPOSITORY=$(jupyter_notebook_image_repository)" \
 		"$(CURDIR)/docker/jupyter"
 
-build-documentation: build-development-image ## Build documentation
+build-documentation:
+
+	@ make build-development-image
+
+	@ make build-documentation-standalone
```

I've learned not so long ago that using `$(MAKE)` is better than calling `make` directly in a Makefile ([docs here](https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html)).

On top, I think it's better to list a target's dependencies as `target: deps`:

```suggestion
build-documentation: build-development-image

	@ $(MAKE) build-documentation-standalone
```